### ISSUE TITLE
Missing repository information in NuGet package

### ISSUE THREAD

**Issue by ggmueller** [JasperFx/marten#3345]
The Marten NuGet-Package does not include the source repository in its metadata:

```xml
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
  <metadata>
    <id>Marten</id>
    <version>7.26.0</version>
    <authors>Jeremy D. Miller,Babu Annamalai,Oskar Dudycz,Joona-Pekka Kokko</authors>
    <owners></owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <license type="expression">MIT</license>
    <projectUrl>https://martendb.io/</projectUrl>
    <iconUrl>https://martendb.io/logo.png</iconUrl>
    <description>.NET Transactional Document DB and Event Store on PostgreSQL</description>
    <dependencies>
    ...
  </metadata>
</package>
```

While e.g. `Marten.Commandline` has this information:

```xml
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
  <metadata>
    <id>Marten.CommandLine</id>
    <version>7.26.0</version>
    <authors>Jeremy D. Miller,Babu Annamalai,Oskar Dudycz,Joona-Pekka Kokko</authors>
    <owners></owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <license type="expression">MIT</license>
    <projectUrl>https://martendb.io/</projectUrl>
    <iconUrl>https://martendb.io/logo.png</iconUrl>
    <description>Command line tooling for Marten</description>
    <repository type="git" url="https://github.com/JasperFx/marten.git" commit="59d2753cb01efeb6b59381e4dd023c8cc632f5e0" />
    <dependencies>
    ...
  </metadata>
</package>
```

The renovate bot seems to use the information to determine which packages should be updated as a group, and thus fails to do it correctly for Marten.CommandLine & Marten.

**Comment by ggmueller** [comment]
I think the issue is, that the information is provided only when the build is run on Windows:

https://github.com/JasperFx/marten/blob/59d2753cb01efeb6b59381e4dd023c8cc632f5e0/src/Marten/Marten.csproj#L59

**Comment by mysticmind** [comment]
Thanks for the details. I will take a look and get this fixed.

**Comment by mysticmind** [comment]
I had a look and the RepositoryUrl is driven by a common `Directory.Build.props` at the root. If you look up NuGet for `Marten`, it does list the repository URL. Besides, if you unzip the nupkg and check the Marten.nuspec, `repository` entry is there.

So this is non an issue, I am closing the same.

**Comment by mysticmind** [comment]
FYI, @jeremydmiller  has fixed it in the recent commit hence 7.26.1 is fine.

**Comment by ggmueller** [comment]
Awesome, and Renovate bot correctly used the new metadata to update that Marten dependencies.
![image](https://github.com/user-attachments/assets/3aabed78-a943-4a52-bf35-e299acea5e09)

Better for lazy me :sunglasses:
